trunner/test_runner.py

```diff
@@ -56,6 +56,7 @@ def run(self):
         for target, tests in self.tests_per_target.items():
             for test_case in tests:
                 test_case.log_test_started()
+                self.runners[target].status('busy')
```

Comment:
Maybe a dictionary of states somewhere in `config.py` that can. In future we want to add some state there may by less work

Or even better - an Enum class with statuses. Then we can simply use status.BUSY, status.FAILED etc

Reply:
I would prefer simpler solution for now. If we add other states, we need to change it here anyway